Files/Views/LayoutModes/GenericFileBrowser.xaml

```diff
@@ -980,7 +983,7 @@
                     x:Uid="sizeColumn"
                     Width="Auto"
                     MinWidth="100"
-                    Binding="{Binding FileSize}"
+                    Binding="{Binding FileSizeDisplay}"
```

Comment:
Perhaps the AutomationProperties name should say "Not Calculated" and the size should remain blank.

Reply:
@yaichenbaum I was unsuccessful in getting automation properties to work with the DataGrid. The screen reader wouldn't read out the name when the column was selected. I had the same issue with trying to add a name to the sync status glyph. If you want, I could switch back and we can try to implement this at a later date (Perhaps when the DataGrid is modernized?).

Reply:
Let's to that and wait on the new DataGrid control to fix this issue.